app/src/main/java/doit/study/droid/topic/TopicModelView.kt

```diff
@@ -0,0 +1,85 @@
+package doit.study.droid.topic
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import doit.study.droid.data.local.QuizDatabase
+import doit.study.droid.data.local.entity.Tag
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+import timber.log.Timber
+import javax.inject.Inject
+
+class TopicModelView @Inject constructor(
```

Comment:
Should be: TopicViewModel

Reply:
It will be fixed in https://github.com/mgolokhov/dodroid/issues/139